common/new-bet.ts

```diff
@@ -23,6 +23,15 @@ import { noFees } from './fees'
 import { addObjects } from './util/object'
 import { NUMERIC_FIXED_VAR } from './numeric-constants'
 
+export type BetInfo = {
+  newBet: Bet
+  newPool?: object
+  newTotalShares?: object
+  newTotalBets?: object
```

Comment:
I think object is not as specific as { [outcome: string]: number }. I would switch this for these three fields.